gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1085,7 +1086,15 @@ void gnc_file_ofx_import (GtkWindow *parent)
 #endif
 
         DEBUG("Opening selected file");
+        // JEAN: Where the ofx file is loaded.
         libofx_proc_file(libofx_context, selected_filename, AUTODETECT);
+        // Now would be a good time to see whether the view has anything in it!
+        gboolean is_empty = gnc_gen_trans_list_empty(gnc_ofx_importer_gui);
```

Comment:
You can't declare variables in the middle of a block in C, that's a C++ thing.  You could move the declaration to the top of the function, but since `is_empty` is used only in the following if statement, just call `gnc_gen_trans_list_empty` as the condition.